source/d3d11/d3d11_device_context.cpp

```diff
@@ -110,6 +110,9 @@ ULONG STDMETHODCALLTYPE D3D11DeviceContext::Release()
 
 		LOG(INFO) << "Destroyed 'ID3D11DeviceContext" << (_interface_version > 0 ? std::to_string(_interface_version) : "") << "' object " << this << ".";
 
+		// clear the trackers
+		clear_drawcall_stats();
```

Comment:
This is not necessary if `com_ptr` are used as described above, since the `delete` will take care of cleanup.

Reply:
Have you checked if removing this causes memory leaks. Because if I read everything right it shouldn't now and this line is safe to remove.

Reply:
Have you checked if removing this causes memory leaks. Because if I read everything right it shouldn't now and this line is safe to remove.